### ISSUE TITLE
h5 to tflite

### ISSUE THREAD

**Issue by HiIcy** [keras-team/keras#13947]
<em>Please make sure that this is a Bug or a Feature Request and provide all applicable information asked by the template.
If your issue is an **implementation question**, please ask your question on [StackOverflow](http://stackoverflow.com/questions/tagged/keras) or [on the Keras Slack channel](https://keras-slack-autojoin.herokuapp.com/) instead of opening a GitHub issue.</em>  

**System information**  
- Have I written custom code (as opposed to using example directory):  
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):  
- TensorFlow backend (yes / no):  
- TensorFlow version: 1.12.0 
- Keras version:  2.2.4
- Python version:  
- CUDA/cuDNN version:  1.9
- GPU model and memory:  tesla

------------------------------------------------
**When I define a model with a custom loss funtion,**,
**I want to use `tflite_toco` to convert it to tflite,but it show the `Unknown loss`**,
**I can understand the why error occur,But I dont't know how to solve it**
**any appreciatful!**

**Comment by jvishnuvardhan** [comment]
@HiIcy In [this gist](https://colab.research.google.com/gist/jvishnuvardhan/37aefa76e13ced2b5eeb7721d9dc9455/untitled1029.ipynb), i tried to show simple TFlite conversion when there is a simple custom metric was included in the model. Please check the gist and let me know whether this resolved your issue. Thanks!
